lms/static/sass/lms-course.scss

```diff
@@ -0,0 +1,11 @@
+@import 'bourbon/bourbon';
+@import 'vendor/bi-app/bi-app-ltr'; // set the layout for left to right languages
+
+@import 'base/reset';
+@import 'base/variables';
+@import 'base/font_face';
+@import 'base/mixins';
+
+//<THEME-OVERRIDE>
```

Comment:
Why is this comment here? The override happens in a separate Sass compile that's in the independent theme dir.